src/ardrive.ts

```diff
@@ -1049,7 +1000,14 @@ export class ArDrive extends ArDriveAnonymous {
 		let uploadEntityResults: ArFSEntityData[] = [];
 		let folderId: FolderID;
 
-		if (wrappedFolder.existingId) {
+		if (wrappedFolder.fileNameConflict) {
+			if (conflictResolution === skipOnConflicts) {
+				// Return empty result on skip
+				return { entityResults: uploadEntityResults, feeResults: uploadEntityFees };
```

Comment:
You might want to return actual empties here just in case someone changes something upstream later on.